src/period/helpers.js

```diff
@@ -6,10 +6,8 @@ export function formatAsISODate(date) {
     }
 
     const y = date.getFullYear();
-    let m = `${date.getMonth() + 1}`;
-    let d = `${date.getDate()}`;
-    m = m.length < 2 ? `0${m}` : m;
-    d = d.length < 2 ? `0${d}` : d;
+    const m = `0${date.getMonth() + 1}`.substr(-2);
+    const d = `0${date.getDate()}`.substr(-2);
```

Comment:
There are a lot more of these things in there somewhere. I've left a lot of the original stuff in place, but we'll have to refactor it later ;)